dae/dae/tools/dae2parquet.py

```diff
@@ -119,7 +119,8 @@ def import_dae_denovo(
         dae_config, genome, annotation_pipeline,
         families_filename, variants_filename,
         family_format='pedigree', output='.', rows=10000,
-        bucket_index=0, defaults={}, study_id=None, filesystem=None):
+        bucket_index=0, defaults={}, study_id=None, filesystem=None,
+        pedigree_df=None):
 
```

Comment:
Here pedigree_df should be required parameter and family_format parameter should be removed...

Processing of the family data should be done before coming to this function and import_dae_denovo should use pedigree dataframe onl